alluxiofs/core.py

```diff
@@ -50,6 +50,47 @@ def __init__(
         test_options=None,
         **kwargs,
     ):
+        """
+        Initializes an Alluxio filesystem on top of underlying filesystem
+        to leveraging the data caching and management features of Alluxio.
+
+        The Alluxio args:
+            etcd_hosts (str, optional): A comma-separated list of ETCD server hosts
+                in the format "host1:port1,host2:port2,...".
+                ETCD is used for dynamic discovery of Alluxio workers.
+                Either `etcd_hosts` or `worker_hosts` must be specified, not both.
+            worker_hosts (str, optional): A comma-separated list of Alluxio worker hosts
+                in the format "host1:port1,host2:port2,...".
+                Directly specifies workers without using ETCD.
+                Either `etcd_hosts` or `worker_hosts` must be specified, not both.
+            options (dict, optional): A dictionary of Alluxio configuration options
+                where keys are property names and values are property values.
+                These options configure the Alluxio client behavior.
+            logger (logging.Logger, optional): A logger instance for logging messages.
+                If not provided, a default logger with the name "AlluxioFileSystem" is used.
```

Comment:
nit: Not sure whether we should change the logger's name to `Alluxiofs` (or other names), since it may interfere with the underneath AlluxioFileSystem logging?

Reply:
Make sense